xla/service/gpu/ir_emission_utils_test.cc

```diff
@@ -1032,5 +1035,45 @@ ENTRY main {
               IsOkAndHolds(true));
 }
 
+TEST_F(IrEmissionUtilsTest, ProtoFingerprintIsDeterministic) {
+  gpu::GpuBackendConfig proto;
+  auto& knobs = *proto.mutable_cudnn_fmha_backend_config()
+                     ->mutable_algorithm()
+                     ->mutable_tuning_knobs();
+  knobs[0] = 1;
```

Comment:
Is testing just two knobs enough to establish determinism? Even with a non-deterministic implementation this will still be correct 50% of the time. Consider adding more, e.g. 10 or so.